From c7dc2360098b1b405e8fcf1d140987191023e2f6 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Fri, 13 Jul 2018 06:54:31 -0600
Subject: [PATCH 08/47] msm: ipa: Remove duplicate definition of
 list_next_entry()

These were moved to list.h, but the upstream change did not account
for this msm-specific use

Change-Id: I23b23906e15f0db5e770a32e43cd665e3c4afae3
Fixes: fce85b081c08c ("list: introduce list_next_entry() and list_prev_entry()")
Signed-off-by: Kevin F. Haggerty <haggertk@lineageos.org>
---
 drivers/platform/msm/ipa/ipa_dp.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/platform/msm/ipa/ipa_dp.c b/drivers/platform/msm/ipa/ipa_dp.c
index e2856cf680e..4a40744ab66 100644
--- a/drivers/platform/msm/ipa/ipa_dp.c
+++ b/drivers/platform/msm/ipa/ipa_dp.c
@@ -18,8 +18,6 @@
 #include "ipa_i.h"
 
 
-#define list_next_entry(pos, member) \
-	list_entry(pos->member.next, typeof(*pos), member)
 #define IPA_LAST_DESC_CNT 0xFFFF
 #define POLLING_INACTIVITY_RX 40
 #define POLLING_MIN_SLEEP_RX 950
-- 
2.17.1

